### PR TITLE
Fix, system.repo don't use curl on fixed pkgrepo.manage

### PR DESCRIPTION
**Pull request by epcim**
Change-Id: Id5b5a44f3dfbbdd60442fd2f273b72557fa9e191

(change will come from Mirantis gerrit once it pass tests)

### CODE REVIEW

File: linux/system/repo.sls [hunk 1]
```diff
@@ -132,6 +140,9 @@ linux_repo_{{ name }}:
   {%- if repo.key_server is defined %}
   - keyserver: {{ repo.key_server }}
   {%- endif %}
+  {%- if repo.key_url is defined and grains['saltversioninfo'] >= [2017, 7] %}
```
Comment (bbinet) [inline linux/system/repo.sls]: `or repo.key_url.startswith('salt://')`

Comment (bbinet) [reply linux/system/repo.sls]: I mean:
`{%- if repo.key_url is defined and (grains['saltversioninfo'] >= [2017, 7] or repo.key_url.startswith('salt://')) %}`

Comment (epcim) [reply linux/system/repo.sls]: well, you are right, can you send one more PR (I will merge it on GH while traveling home)

Comment (bbinet) [reply linux/system/repo.sls]: See PR #156

**Pull request merged**